src/main/kotlin/org/jetbrains/grammarkit/tasks/GenerateParserTask.kt

```diff
@@ -83,19 +85,17 @@ abstract class GenerateParserTask @Inject constructor(
      */
     @get:InputFiles
     @get:Classpath
-    abstract val classpath: ConfigurableFileCollection
+    abstract val grammarKitClasspath: ConfigurableFileCollection
```

Comment:
Hmmm actually... isn't it a breaking change for the users? 🤔 can't this `val` be assigned from `tasks.register<GenerateParserTask> { ... }` block from end-user code? Same for `GenerateLexerTask`.

Actually... given that now both tasks are `JavaExec` tasks, maybe you can skip dealing with `classpath` completely? Leaving it up to user to set the `classpath` directly, if they wish so 🤔

Reply:
Oh you're right! This val is seen by the user!